### DISCUSSION TITLE
Change notice with a different SDK version

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#926] [Q&A] [answered]
An notice in accordance with SDK 1.7 was published via our platform. A corresponding change notice should be published with SDK 1.10. However, due to changes to the validation rules, a field (BT-610) is mandatory in SDK 1.10, which was not yet mandatory in SDK 1.7. However, the corresponding field can no longer be changed in the change notice. How should we deal with this?

**Comment by YvesJo** [comment, marked as answer]
Hi,
This should normally be addressed via the helpdesk.
The notice has to comply with the version it is created with. When the content of the change notice can not be modified anymore and when it is still time, you may cancel its publication and redo the change. When the Change publication may not be cancelled anymore, you may apply a second change to fix what has to be corrected or submit a change notice to cancel the previous change notice and redo a change containing all the changes.

**Reply by JanEmde** [reply]
Thanks a lot!

The problem with the helpdesk is that the answer sometimes takes 4-6 weeks....

**Reply by YvesJo** [reply]
I had a short exchange with a colleague lately and would like to add the following.
When it comes to a Cancel Change notice, the notice should be submitted using the same SDK the notice to be declared void was initially submitted with. The reason is that, the latest SDK may have rules which would require to update the notice content.
